faq.md

```diff
@@ -68,17 +53,13 @@ There are different ways partners can contribute to this project:
 * Build the infrastructure to support OPEA projects 
 
 ## Where can partners see the latest draft of the Conceptual Framework spec? 
-A version of the spec is available in the docs repo in this project 
+A version of the spec is available in the documentation repository within this project. 
```

Comment:
The name of the repo is "docs", so we should include mentioning this.

Reply:
added "docs" and link to repo